File: read_pdf.js
const fs = require('fs');
const pdf = require('pdf-parse');

let dataBuffer = fs.readFileSync('c:/Users/onurb/Desktop/onurburak-su.github.io-main/onurburak-su.github.io-main/WebLab_Flexbox_Uygulama.pdf');

pdf(dataBuffer).then(function(data) {
    console.log(data.text);
}).catch(err => {
    console.error(err);
});

File: read_pdf.py
import sys
import subprocess

try:
    import pypdf
except ImportError:
    subprocess.check_call([sys.executable, "-m", "pip", "install", "pypdf"])
    import pypdf

reader = pypdf.PdfReader('c:/Users/onurb/Desktop/onurburak-su.github.io-main/onurburak-su.github.io-main/WebLab_Flexbox_Uygulama.pdf')
for page in reader.pages:
    print(page.extract_text())
